tox.ini

```diff
@@ -1,11 +1,61 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
 [tox]
-envlist = pytest{45,6,7,master}-py{37,38},py27-pytest{35, 45}
-skip_missing_interpreters = true
+envlist = lint,black,isort
```

Comment:
tests?

Reply:
There were no tests directly here, but https://github.com/datarobot/pytest-stereo/ runs its tests using this plugin. I assumed that was intentional 🙈